.ci/all.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+# Attention, there is no "-x" to avoid problems on Travis
+set -e
+
+# result of command goes to $RUN_CI
+# 0 = skip
+# 1 = no skip
+function should_run_ci {
```

Comment:
we need to fix terminology .....
CI is whole virtual environment.
This method to decision not for whole CI .... it does decision for certain command/build-item execution.

what about `should_run_command` ? `should_run_job` ?

Reply:
Done.
I renamed a lot of other things to follow the pattern.